setup.py

```diff
@@ -87,7 +87,7 @@
     'beamforming', 'cross correlation', 'database', 'dataless',
     'Dataless SEED', 'datamark', 'earthquakes', 'Earthworm', 'EIDA',
     'envelope', 'ESRI', 'events', 'FDSN', 'features', 'filter',
-    'focal mechanism', 'GSE1', 'GSE2', 'hob', 'Tau-P', 'imaging',
+    'focal mechanism', 'GSE1', 'GSE2', 'GCF', 'hob', 'Tau-P', 'imaging',
```

Comment:
I'd like to say "please keep in alphabetical order", but `Tau-P` is completely in the wrong spot..